src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -145,8 +143,8 @@ public FleetStatusService(Topics topics, MqttClient mqttClient, DeploymentStatus
         this.publisher = new MqttChunkedPayloadPublisher<>(this.mqttClient);
         this.architecture = platformResolver.getCurrentPlatform()
                 .getOrDefault(PlatformResolver.ARCHITECTURE_KEY, PlatformResolver.UNKNOWN_KEYWORD);
-        this.periodicUpdateIntervalSec = TestFeatureParameters.retrieveWithDefault(Double.class,
-                FLEET_STATUS_TEST_PERIODIC_UPDATE_INTERVAL_SEC, periodicUpdateIntervalSec).intValue();
+        this.periodicPublishIntervalSec = TestFeatureParameters.retrieveWithDefault(Double.class,
+                FLEET_STATUS_TEST_PERIODIC_UPDATE_INTERVAL_SEC, periodicPublishIntervalSec).intValue();
```

Comment:
If the previous deployment changes the config to a smaller value, it seems the smaller value will be kept in config but not affecting the FSS publish. Is it correct?
In this case, if nucleus restarts, will FSS pick up the smaller value here and actually use it for publish?